### ISSUE TITLE
how would we do auto-update?

### ISSUE THREAD

**Issue by bypassiwastaken** [bypassiwastaken/skiovox-helper#19]
i know how to set up auto-update. i just want people's recommendations on how to host this and whether its worth it. skiovox.com seems reasonable enough.

however signing the extension and editing the xml file manually would be a crap ton of work. also i lost the private key so we'd need a new id.

**Comment by nicolasog13** [comment]
maybe use skiovox and put it as like skiovox.com/upd or /update
you could also have another shortlink that takes you to the github with something like skiovox.com/repo

**Comment by bypassiwastaken** [comment]
right. the only issue with this is that it would be a new extension id. given that you cant really delete extensions, i find this very cringe.

**Comment by zkayns** [comment]
maybe don't auto update and just notify the user that a new update is available?

**Comment by bypassiwastaken** [comment]
ok so apparently you can remove old extensions so there is a possibility that we could make a version of the extension which would auto update. HOWEVER removing the old one is a bit of a hassle

**Comment by nicolasog13** [comment]
> right. the only issue with this is that it would be a new extension id. given that you cant really delete extensions, i find this very cringe.

you can delete extensions from the little puzzle icon on the top right

**Comment by Mazedotexe** [comment]
> > right. the only issue with this is that it would be a new extension id. given that you cant really delete extensions, i find this very cringe.
> 
> you can delete extensions from the little puzzle icon on the top right

not exactly what he meant lol

**Comment by Mazedotexe** [comment]
> maybe don't auto update and just notify the user that a new update is available?

yes, perfect idea.  bypassi, please just do this.

**Comment by Honkychunkymonkey** [comment]
> > maybe don't auto update and just notify the user that a new update is available?
> 
> yes, perfect idea.  bypassi, please just do this.

Or just ask the user if they'd like to auto update or not? If they decline, they can still do it manually
